spec/controllers/omniauth_callbacks_controller_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+require 'byebug'
+
+# RSpec.describe Users::OmniauthCallbacksController, type: :controller do
```

Comment:
Commented out these test as we are merging this work from another branch. @200455939-yashu

Reply:
this is resolved in the PR #869